.circleci/config.yml

```diff
@@ -67,6 +67,19 @@ jobs:
       - run: docker build . -f ./build/Dockerfile-rhel -t $OAO_IMAGE_RHCC:$CIRCLE_TAG
       - run: docker push $OAO_IMAGE_RHCC:$CIRCLE_TAG
 
+  deploy-release-snapshot:
+    <<: *defaults
+    steps:
+      - setup_remote_docker
+      - run: |
+          mkdir -p ~/.docker && chmod 0700 ~/.docker
+          touch ~/.docker/config.json && chmod 0600 ~/.docker/config.json
+          base64 -d >~/.docker/config.json <<<"$OAO_DOCKER_AUTH"
+          export RELEASE_TAG=${CIRCLE_TAG//release-/v}-snapshot
+      - attach_workspace:
+          at: .
+      - run: docker build . -f ./build/Dockerfile -t $OAO_IMAGE:$RELEASE_TAG
```

Comment:
And as a final confirmation, are we sure that RELEASE_TAG is available here? Since they are two different `run` sections. Not sure how CircleCI handles this.

Reply:
no it does not.
tested.

Reply:
wrong env var, fixed now.